src/main/webapp/dev/js/common/instructorEnroll.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Retrieves updated column header order and generates a header string.
+ *
+ * Example: Changes this array ['Section', 'Team', 'Name', 'Email', 'Comments']
+ * into a string = "Section|Team|Name|Email|Comments\n"
+ *
+ * @returns {string} updated header string
+ */
+function getUpdatedHeaderString(handsontableColHeader) {
+    const colHeaders = handsontableColHeader.join('|');
+    return colHeaders.concat('\n');
+}
+
+/**
+ * Retrieves user data rows rows in the spreadsheet interface and transforms it into a string.
+ *
+ * Null value from cell is changed to empty string after .join(). Filters empty rows in the process.
+ *
+ * Example:
+ * 2 by 5 spreadsheetData (before)
+ * ['TestSection1', 'Team1', 'null', 'test1@xample.com', 'test1comments']
+ * ['TestSection2', null, 'TestName2', 'test2@example.com', null]
+ *
+ * 2 by 5 spreadsheetData (after)
+ * "TestSection1|Team1||test1@xample.com|test1comments\n
+ *  TestSection2||TestName2|test2@example.com|\n"
+ *
+ * @returns {string} user data rows
+ */
+function getUserDataRows(spreadsheetData) {
+    // needs to check for '' as an initial empty row with null values will be converted to e.g. "||||" after .map
+    return spreadsheetData.filter(row => (!row.every(cell => cell === null || cell === '')))
+            .map(row => row.join('|'))
+            .join('\n');
+}
+
+/**
+ * Pushes data from spreadsheetDataRows into an array.
```

Comment:
Minor nit: Using the word `push`, `into an array` is not very appropriate. Also I don't like the name `getUpdatedData`, too generic.